tests/resources/db/latest_schema.sql

```diff
@@ -166,7 +166,7 @@ CREATE TABLE model_version_tags (
 
 CREATE TABLE params (
 	key VARCHAR(250) NOT NULL,
-	value VARCHAR(8000) NOT NULL,
+	value VARCHAR(500) NOT NULL,
```

Comment:
This seems concerning. Don't know why this change was made by `./tests/db/update_schemas.sh`.

Reply:
Alright, I manually overrode those changes